tests/integration/integration_test.go

```diff
@@ -186,6 +186,52 @@ func checkSecurityFileOpenExecve(t *testing.T, gotOutput *[]trace.Event) {
 		assert.Equal(t, events.Execve, syscall)
 	}
 }
+
+func checkScope42SecurityFileOpenLs(t *testing.T, gotOutput *[]trace.Event) {
+	_, err := forkAndExecFunction(doLs)
+	require.NoError(t, err)
+
+	waitForTraceeOutput(t, gotOutput, time.Now(), true)
+
+	for _, evt := range *gotOutput {
+		// ls - scope 42
+		assert.Equal(t, "ls", evt.ProcessName)
+		assert.Equal(t, uint64(1<<41), evt.MatchedScopes)
+		arg, err := helpers.GetTraceeArgumentByName(evt, "pathname")
+		require.NoError(t, err)
+		assert.Contains(t, arg.Value, "integration")
+	}
+}
+
+func checkScopes4n2Execve(t *testing.T, gotOutput *[]trace.Event) {
```

Comment:
typo in function name?

Reply:
not quite, but it can be clearer:

```suggestion
func checkExecveOnScopes4And2(t *testing.T, gotOutput *[]trace.Event) {
```